.pre-commit-config.yaml

```diff
@@ -52,6 +52,14 @@ repos:
         files: \.py$  # only target python files, for example - skip contract_registry.json
         args: [--license-filepath, '.nucypher-license', --detect-license-in-X-top-lines=16, --comment-style, '"""||"""']
 
+  - repo: https://github.com/akaihola/darker
+    rev: 1.4.2
+    hooks:
+    -   id: darker
+        args: [--isort]
```

Comment:
Any thoughts on configuring line length - the default is `88`?

Reply:
I've never had a problem with 88 - i know monitors have got a lot bigger, but i regularly find myself working on laptops/opening multiple files side by side/looking at diffs on github.

Whatever we choose, someone is going to be unhappy! 😁

Reply:
Haha fair enough 😄 .